app/models/resource_action.rb

```diff
@@ -20,8 +20,7 @@ def automate_queue_hash(target, override_attrs, user, open_url_task_id = nil)
     elsif target.kind_of?(Hash)
       override_values = target
     elsif target.kind_of?(Array) || target.kind_of?(ActiveRecord::Relation)
-      klass = target.first.id.class
-      object_ids = target.collect { |t| "#{klass}::#{t.id}" }.join(",")
+      object_ids = target.collect { |t| "#{t.class.name.demodulize}::#{t.id}" }.join(",")
```

Comment:
We should be using Automate engine methods here.  See https://github.com/ManageIQ/manageiq-automation_engine/blob/master/lib/miq_automation_engine/engine/miq_ae_engine.rb#L162-L190